alerts/plugins/port_scan_enrichment.py

```diff
@@ -196,9 +196,7 @@ def enrich(alert, search_fn, search_window, max_connections):
         for hit in results.get('hits', [])
     ]
 
-    alert['details'] = {
-        'recentconnections': []
-    }
+    alert['details']['recentconnections'] = []
```

Comment:
I think we also want to add a check to see if alert['details'] exists beforehand, and if it doesn't, we set it.